spec/services/export/s3_downloader_spec.rb

```diff
@@ -0,0 +1,82 @@
+require "rails_helper"
+
+RSpec.describe Export::S3Downloader do
+  let(:response_body) { double("response_body", read: double) }
+  let(:response) { double("response", body: response_body) }
+  let(:aws_credentials) { double("aws credentials") }
+  let(:aws_client) { instance_double(Aws::S3::Client, get_object: response) }
+
+  let(:s3_bucket) { double("s3 bucket") }
+  let(:s3_uploader_config) {
+    instance_double(
+      Export::S3UploaderConfig,
+      key_id: "key id",
+      secret_key: "secret key",
+      region: "region",
+      bucket: s3_bucket
+    )
+  }
+
+  let(:filename) { "Q1_report0101202312345.csv" }
+
+  let(:public) { true }
```

Comment:
I don't think we're using this

Reply:
I think that's in a different spec?

Reply:
hah, I've just come across this in my renaming. You're right! Removing it now.